tests/test_api_endpoints.py

```diff
@@ -0,0 +1,219 @@
+import os
+import json
+import base64
+import psycopg2
+import requests
+from contextlib import closing
+
+BASE_URL = "http://localhost:5001"
```

Comment:
Only one port, 5001 here. Needs to be 5000

Reply:
It is testing rust that runs the api on port 5001. I can change it but it is going to fail since this PR doesn't include the refactor.